sidebars.yaml

```diff
@@ -313,6 +314,8 @@
                 - page: docs/references/protocol/data-types/base58-encodings.md
                 - page: docs/references/protocol/data-types/currency-formats.md
                 - page: docs/references/protocol/data-types/nftoken.md
+                - page: docs/references/protocol/data-types/mptoken.md
```

Comment:
Should these be under Ledger Data Types > Ledger Entry Types?

Reply:
Moved.